lib/modules/manager/flux/common.ts

```diff
@@ -1,6 +1,6 @@
 import { regEx } from '../../../util/regex';
 
-export const systemManifestRegex = '(^|/)flux-system/gotk-components\\.yaml$';
+export const systemManifestRegex = '(^|\\/)flux-system\\/(?:.+\\/)?gotk-components\\.yaml$';
 export function isSystemManifest(file: string): boolean {
```

Comment:
```suggestion

 export function isSystemManifest(file: string): boolean {
```
this probably needs a newline